topomodelx/nn/cell/can.py

```diff
@@ -47,14 +47,14 @@ def __init__(
         in_channels_0,
         in_channels_1,
         out_channels,
-        num_classes,
         dropout=0.5,
-        heads=3,
+        heads=2,
```

Comment:
The documentation for `heads` in the docstring needs to be updated as well.

Reply:
Done, thank you.